docs/HOWTO.md

```diff
@@ -405,9 +405,9 @@ autosklearn_oldgen:
   description: "this will use the latest autosklearn version from the old generation"
 
 H2OAutoML_nightly:
-  extends: H2OAutoML
-  version: 'latest'
-  description: "this will use the nightly H2O build if there's no H2O version already installed (use --setup=force to refresh the version)"
+  module: frameworks.H2OAutoML
+  setup_cmd: 'LATEST_H2O=`curl http://h2o-release.s3.amazonaws.com/h2o/master/latest` && pip install --no-cache-dir -U "http://h2o-release.s3.amazonaws.com/h2o/master/${{LATEST_H2O}}/Python/h2o-3.29.0.${{LATEST_H2O}}-py2.py3-none-any.whl"'
+  version: 'nightly'
```

Comment:
sorry for that, I simplified this by supporting the `latest` version in h2o's `setup.sh`, but you're right, it's probably better to keep it to document the `setup_cmd` param.

Reply:
No problem :) Agreed that for the actual use case there is a better way to do it (now), but for documentation purposes we need _something_.